community/index/src/test/java/org/neo4j/index/gbptree/GBPTreeIT.java

```diff
@@ -498,6 +498,196 @@ else if ( lastSeenKey > nextToSee )
     }
 
     @Test
+    public void shouldReadCorrectlyWhenConcurrentlyInsertingOutOfOrderAndSeekingBackwards() throws Throwable
```

Comment:
comments for this test is applicable to his copy pasted brother bellow as well :)